crates/oxc_linter/src/rules/import/no_absolute_path.rs

```diff
@@ -155,6 +155,7 @@ fn check_path_is_absolute(path_str: &str) -> bool {
 }
 
 #[test]
+#[cfg(not(windows))]
```

Comment:
Can you provide the context for this?

Reply:
yeah sure, Path::is_absolute has a different impl on windows vs posix which means these tests fail on windows. i'll add a comment in the code